src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentBindLoweringPass.cs

```diff
@@ -212,7 +212,7 @@ private IntermediateNode[] RewriteUsage(IntermediateNode parent, TagHelperProper
 
             var valueExpressionTokens = new List<IntermediateToken>();
             var changeExpressionTokens = new List<IntermediateToken>();
-            if (changeAttribute == null || changeAttribute.IsDelegateProperty())
```

Comment:
This subtle change makes *delegate bind* the special case instead of the default case. Now `bind-...` to elements prefers `EventCallback<>`